python/lsst/ts/externalscripts/auxtel/correct_pointing.py

```diff
@@ -325,6 +325,12 @@ async def _center(self) -> float:
         )
 
         offset_x, offset_y = await self.find_offset(image_id=acquisition_image_ids[0])
+
+        if np.isnan(offset_x) | np.isnan(offset_y):
```

Comment:
I think to compare the two booleans returned by `np.isnan` it is best to use the `or` operator instead of the bitwise `|` comparison. Even if they yield the same results, at the very least, using `or` is more readable.